src/MCP2515/MCP2515_Control.cpp

```diff
@@ -11,11 +11,10 @@
 
 #include "MCP2515_Control.h"
 
-#include <Arduino.h>
-
 #undef min
 #undef max
 #include <algorithm>
+#include <cstring>
```

Comment:
For which function do we need this include? Remove, if not necessary.

Reply:
cstring is required for memcpy